static/templates/settings/settings_save_discard_widget.hbs

```diff
@@ -19,4 +18,3 @@
     </div>
     <div class="inline-block subsection-failed-status"><p class="hide"></p></div>
 </div>
-{{/if}}
```

Comment:
Why do you need this change?

Reply:
I am using the settings_save_discard_widget at the user settings page side to. So to make the widget visible to the user.